presentation/src/main/java/com/android254/presentation/about/view/AboutViewModel.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.presentation.about.view
+
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import com.android254.presentation.models.OrganizingTeamMember
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class AboutViewModel @Inject constructor() : ViewModel() {
+
+    val sampleImageUrl: String = "https://media-exp1.licdn.com/dms/image/C4D03AQGn58utIO-x3w/profile-displayphoto-shrink_200_200/0/1637478114039?e=2147483647&v=beta&t=3kIon0YJQNHZojD3Dt5HVODJqHsKdf2YKP1SfWeROnI"
+
+    val organizingTeamMembers = mutableStateOf(
+        listOf(
+            OrganizingTeamMember(
+                name = "Frank Tamre",
+                desc = "Main Man",
+                image = sampleImageUrl
```

Comment:
Since this is sample data just create one item and repeat it ie.
```
val organizingTeamMembers = MutableList(n) {_->OrganizingTeamMember(
                name = "Marvin Collins",
                desc = "Main man 2",
                image = sampleImageUrl
            )}
```

Reply:
@chepsi I have updated this..thank you